README.md

```diff
@@ -220,13 +220,19 @@ danger-swift command --danger-js-path path/to/danger-js
 
 #### Current working directory
 
-Many people prefers using the SPM Danger configuration, because is more performing.
-But having a `Package.swift` on your root folder can be annoying, especially now that Xcode (from 11) doesn't put on the recents list an `xcproj` (or `xcworkspace`) when there is a `Package.swift` in the same folder.
+Many people prefer using the SPM Danger configuration because is more performant.
+But having a `Package.swift` in your root folder can be annoying, especially now that Xcode (from 11) doesn't put an `xcproj` (or `xcworkspace`) on the recents list when there is a `Package.swift` in the same folder.
 With the `--cwd` parameter you can specify a working directory.
-This allows you to have your `Package.swift` in another directory and still run danger-swift as it was executed from your project root directory.
+This allows you to have your `Package.swift` in another directory and still run danger-swift as it was executed from your project root directory. Since you no longer have a `Package.swift` in your root directory, you'll also have to specify that package path to `swift run`.
 
 ```swift
-swift run danger-swift command --cwd path/to/working-directory
+swift run --package-path path/to/working-directory danger-swift command --cwd path/to/working-directory
```

Comment:
`--package-path` and `--cwd` do almost the same thing, but in different ways (so you shouldn't need `--cwd` if you have `--package-path`), but `--package-path` doesn't work in all the cases, it has still some corner cases that needs to be handled, and this is why it is not suggested yet, once it fully supported cwd should be deprecated.

The correct command should be `cd path && swift run danger ci --cwd rootFolder`